src/Controller/Helpers/SchemaHelper.php

```diff
@@ -205,4 +208,29 @@ public function buildSchemaForContributor(Contribution $contribution): array
         }
         return $schema;
     }
+
+    public function buildSchemaForImage(Image $image, Programme $programme)
+    {
+        $schema = [
+            "@type" => "ImageObject",
+            "caption" => $programme->getTitle() . ' - ' . $image->getTitle() . ' - ' . $image->getLongestSynopsis(),
+            "thumbnail" => "https://ichef.bbci.co.uk/images/ic/224xn/" . $image->getPid() . "jpg",
+            "contentUrl" => $image->getUrl(560),
```

Comment:
```suggestion
            "contentUrl" => $image->getUrl(976),
```
I suggest to match the size of the image we display

Reply:
Not sure if there is any reason to keep it as 560 or just got missed when fixing something else.